src/EditorFeatures/Core/PublicAPI.Shipped.txt

```diff
@@ -1,8 +1,2 @@
 Microsoft.CodeAnalysis.Editor.Peek.IPeekableItemFactory
 Microsoft.CodeAnalysis.Editor.Peek.IPeekableItemFactory.GetPeekableItemsAsync(Microsoft.CodeAnalysis.ISymbol symbol, Microsoft.CodeAnalysis.Project project, Microsoft.VisualStudio.Language.Intellisense.IPeekResultFactory peekResultFactory, System.Threading.CancellationToken cancellationToken) -> System.Threading.Tasks.Task<System.Collections.Generic.IEnumerable<Microsoft.VisualStudio.Language.Intellisense.IPeekableItem>>
-Microsoft.CodeAnalysis.Editor.Tags.ExportImageMonikerServiceAttribute
```

Comment:
do we need to discuss this breaking change with anyone?  or is the presumption that it was never ok for anyone to use this but us?

Reply:
we just don’t know of anyone who depends on it, so we don’t know who to discuss this with

Reply:
Do F# or TS use it?

Reply:
nope, we checked http://index and grep.app

Reply:
And yes, this was _OK_ to use as a public API, it's just that as best we can tell nobody ever needed it in the end.

Reply:
wfm